code/game/centcomm_orders/orders_service.dm

```diff
@@ -67,6 +67,12 @@
 			)
 			worth = 700*requested[requested[1]]
 			sauce = 2
+		if (8)
+			requested = list(
+				/obj/item/weapon/reagent_containers/food/snacks/sliceable/pizza/blingpizza = rand(2,5)
+			)
+			name = "Central Command's Executive Suite" 
+			worth = 800*requested[requested[1]]
```

Comment:
big reward for something you can get by ordering pizza crates for 75...

Reply:
@Eneocho gotta reach the top monthly richest escapee somehow

Reply:
I want them available on the pizza cratees, so I'll just remove the request, to be honest I didn't even think about that.

Reply:
There, removed.